lang-guide/chapters/types/basic_types/duration.md

```diff
@@ -1,72 +1,47 @@
 # Duration
 
-What it is: A value representing a unit of passage of time.
-
-Annotation: `duration`
+- **_Description:_** Represent a unit of a passage of time
+- **_Annotation:_** `duration`
+- **_Literal Syntax:_** A numeric (integer or decimal) literal followed immediately by a duration unit (listed below). E.g., `10sec`, `987us`, `-34.65day`
+- **_Casts:_** [`into duration`](/commands/docs/into_duration.md)
 
 Durations are internally stored as a number of nanoseconds.
 
-## Literals
-
-to form a Duration literal you need to combine a numeric value with a a unit of time.
-The numeric literal part must be a signed integer or floating point number literal.
-The unit part must be one of a specific set of strings listed below.
-
-```
-<number><unit>
-
-# E.g.
-
-10sec
-987us
--34.65day
-
-```
-
 This chart shows all duration units currently supported:
 
-| Duration | Length          |
-| -------- | --------------- |
-| `1ns`    | one nanosecond  |
-| `1us`    | one microsecond |
-| `1ms`    | one millisecond |
-| `1sec`   | one second      |
-| `1min`   | one minute      |
-| `1hr`    | one hour        |
-| `1day`   | one day         |
-| `1wk`    | one week        |
+| Duration  | Length          |
+| --------- | --------------- |
+| `ns`      | one nanosecond  |
```

Comment:
The headers need to be changed if you plan on removing all the 1's. They appear to no longer be Duration and Length but Abbreviation and Definition.

Reply:
Resolved in latest commit